src/brogue/Rogue.h

```diff
@@ -2882,7 +2882,7 @@ extern "C" {
 
     creature *generateMonster(short monsterID, boolean itemPossible, boolean mutationPossible);
     short chooseMonster(short forLevel);
-    creature *spawnHorde(short hordeID, short x, short y, unsigned long forbiddenFlags, unsigned long requiredFlags);
+    creature *spawnHorde(short hordeID, short x, short y, unsigned long forbiddenFlags, unsigned long requiredFlags, boolean periodic);
```

Comment:
I'm having trouble intuiting the meaning of `periodic`.  Can you explain further what a periodic horde vs a non-periodic horde is?

Reply:
Periodic is for the additional spawns when you linger on a level.